include/ofi_recvwin.h

```diff
@@ -50,7 +50,7 @@
 #include <ofi_rbuf.h>
 
 #define OFI_DECL_RECVWIN_BUF(entrytype, name, id_type)			\
-OFI_DECLARE_CIRQUE(entrytype, recvwin_cirq);				\
+OFI_DECLARE_CIRQUE(entrytype, recvwin_cirq)				\
```

Comment:
The OFI_DECLARE_CIRQUE should be changed, so that the caller can add the ';' at the end.  That would be the most natural usage